base/steer/FairRun.h

```diff
@@ -154,6 +155,12 @@ class FairRun : public TNamed
     //** Mark/Unmark event to be filled into output. Default is TRUE. */
     void MarkFill(Bool_t flag) { fMarkFill = flag; }
 
+    //** Get option string */
+    TString GetOptoins(){return fOptions;};
+
+    //** Set option string */
+    void SetOptions(TString s){fOptions=s;};
```

Comment:
Yes, `TString` has short string optimization, but I believe, it is still better to pass `const TString&` instead of by value.